docs/ENVIRONMENT_SETUP.md

```diff
@@ -0,0 +1,53 @@
+# OctoFarm Server Environment Variables

```

Comment:
Environment documentation. I dont like it like this, but its better than nothing.
The reason I dont like it is because it doesnt look like a 'list of variables', it looks more like a wall of text to me.

Can improve with different markup, a table of contents (toc) or a table to enumerate. Ideas?
Comparison: https://docs.octoprint.org/en/master/configuration/config_yaml.html